### DISCUSSION TITLE
[일지] 6월 1일 (화) 오후 회의록

### DISCUSSION THREAD

**Discussion by olenmg** [bcaitech1/p4-opt-5-vibrhanium-#33] [일지]
### 모델링
- optuna부분 commit은 오늘 끝?

- logging 필요한 부분
    - metric에서 F1, MACs 
    - model yaml, hyperparameter yaml

- 고정 설정
    - optimizer: AdamW
    - lr scheduler: CosineAnnealingLR
    - Loss: CrossEntropyLoss
    - Batch size: 64
    - epoch: 100
    - learning rate: 1e-4
    - input image size: 224
    - input image size는 좀 더 고민

- `train_optuna.py`와 `train.py`의 WandB logging을 다르게?
    - `trainer.py`에서 분기? 나누기(optuna인지 그냥인지에 따라서)

- RDB 연결 [링크](https://optuna.readthedocs.io/en/stable/tutorial/20_recipes/001_rdb.html#rdb)

### Special mission 5, 6
- RDB를 AWS에서 열고 연결 예정
    - RDB 연결 해보고싶은 사람은 슬랙에 뿌려준 링크로 연결해보기
- WandB 연결은 mission 3, 4 완료 후 진행 예정

### 저번주 칸반보드 완료된거 닫기
- 완료

### 멘토님께 드릴 질문
- 로봇 프로젝트
    - 모델 플래시 메모리 크기가 32kb인데 이정도에 올릴만한 태스크가 있나요?(아주 간단한거라도... 가능하다고 해주세요)
    - 어느정도 작업까지 가능할지?
    - 노래도 할 수 있나요..? ^^
    - 로봇 위치 고정시켜놓는다는 전제하에 모델 개발해도 될 듯
    - 활용할 데이터는 고민해보기

- pretrained된 가중치를 사용할 때는 모델의 모양이 맞아야 가져올 수 있는 것으로 알고 있습니다. 그 가중치를 가져온 모델 구조에서 일부 블록을 빼내어 파라미터를 줄이면서 기존 pretrained된 가중치의 일부를 사용할 수 있는지 궁금합니다!

- 네이버 채용 단계 때 준비는 어떤 방식으로 하셨나요?

### 그 외 
- 목요일 4시로 멘토링 시간 조정
